core/shared/src/main/scala/scalaz/zio/package.scala

```diff
@@ -4,12 +4,7 @@ package scalaz
 
 package object zio {
 
-  implicit class IOVoidSyntax[A](val io: IO[Void, A]) extends AnyRef {
-    def apply[E]: IO[E, A]      = io.asInstanceOf[IO[E, A]]
-    def widenError[E]: IO[E, A] = apply
-  }
-
-  implicit class IOSyntax[E, A](val io: IO[E, A]) extends AnyRef {
+  implicit class IOSyntax[+E, A](val io: IO[E, A]) extends AnyRef {
```

Comment:
I'm pretty sure this one can be deleted now. You can imply omit `widenError` any place it is called.